### ISSUE TITLE
CSV Reader cant open account-combos.csv

### ISSUE THREAD

**Issue by niklas2110** [maxtheaxe/twitch-drop-farmer#1]
Hey there,
I just created the csv file for my usernames and csv reader cant find the file/path. Any known issue with that, I couldt find a fix for it online. 
I know its a year old but maybe you can look into it :)

**Comment by maxtheaxe** [comment]
hey there, have you made sure that the csv is in the correct location? it should be in the same folder you've cloned the directory into (alongside the python file you run)

**Comment by niklas2110** [comment]
yeah its in the same folder and it cant find it. Ive been messing around with the csv reader but I couldt figure it out

**Comment by maxtheaxe** [comment]
can you send me a screenshot of your console and your folder, please? we'll figure this out
